packages/frontend/src/components/NewsDrawer/NewsItem.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useMemo } from 'react'
+import { BiSolidRocket } from 'react-icons/bi'
+import { Box, Image, Text } from '@chakra-ui/react'
+import { Badge, BadgeLeftIcon } from '@opengovsg/design-system-react'
+import MarkdownRenderer from 'components/MarkdownRenderer'
+import { NEWS_DRAWER_COMPONENT } from 'components/MarkdownRenderer/Components/NewsDrawerComponent'
+import { format } from 'date-fns'
+import { AnimationConfigWithData } from 'lottie-web'
+import { RequireExactlyOne } from 'type-fest'
+
+import LottieWebAnimation from './LottieWebAnimation'
+
+export type NewsItemMultimedia = RequireExactlyOne<
+  {
+    url: string
+    alt: string
+    animationData: AnimationConfigWithData['animationData']
+  },
+  'url' | 'animationData'
+>
+export interface NewsItemProps {
+  date: Date
+  isNewFeature?: boolean
+  title: string
+  details: string
+  multimedia?: NewsItemMultimedia
+}
+
+const DATE_FORMAT = 'dd MMM yyyy'
+
+export default function NewsItem(props: NewsItemProps) {
+  const { date, isNewFeature = false, title, details, multimedia } = props
+  const formattedDate = format(date, DATE_FORMAT)
+
+  const displayedMultimedia = useMemo(() => {
+    if (!multimedia) {
+      return
+    }
+    if (multimedia.animationData) {
+      return (
+        <LottieWebAnimation
+          title={multimedia.alt}
+          animationData={multimedia.animationData}
+        ></LottieWebAnimation>
+      )
+    }
+    return (
+      <Image
+        fit="fill"
+        src={multimedia.url}
+        title={multimedia.alt}
+        alt={multimedia.alt}
+      />
+    )
+  }, [multimedia])
+
+  return (
+    <Box>
+      <Text textStyle="caption-1" color="secondary.400">
+        {formattedDate}
+      </Text>
+      {isNewFeature && (
```

Comment:
do u think instead of manually specifying `isNewFeature`, we can check if the date is within the last month.

Reply:
sounds ok to me, ill check within 30 days from the current date instead? Since a news item should still be new if its on 31st Aug even though the current day could be 1st Sep for example